packages/react-instantsearch/src/templates/Carousel.tsx

```diff
@@ -0,0 +1,36 @@
+import {
```

Comment:
should this folder be `components`?

Reply:
Agree with @Haroenv, we have different APIs in InstantSearch.js (`templates` object) and React InstantSearch (individual component props) for customizing the UI so it makes sense to have different names.